portal/src/main/webapp/js/src/crosscancer.js

```diff
@@ -85,28 +85,64 @@
 
         var isThereHetLoss = false;
         var isThereGain = false;
-        var filterAndSortData = function(histDataOrg) {
+        var filterAndSortData = function(histDataOrg, sliderValue, metaData, totalSamSliderValue) {
+            var threshold = 0;
+            if(sliderValue !== undefined && sliderValue !== null){
```

Comment:
You probably also want to make sure it is a number. You can just use ‘_.isNumber(sliderValue)'  instead of 'sliderValue !== undefined && sliderValue !== null'